cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -679,6 +679,183 @@ const char* gdv_fn_upper_utf8(int64_t context, const char* data, int32_t data_le
   return out;
 }
 
+GDV_FORCE_INLINE
+gdv_uint64 unsignedLongDiv(gdv_int64 x, gdv_int32 m) {
```

Comment:
Change the variables that are using **gdv_uint64** to uint64_t and for the other too. We just use **gdv_name** when creating macros to make our life easy.

In other cases when you add the plain name more readable.

Reply:
Change the function name mode to **snake_case**. Gandiva code just uses CamelCase when it is a method inside a class.

Reply:
Done